spec/features/creating_a_new_document_spec.rb

```diff
@@ -0,0 +1,143 @@
+require "spec_helper"
+
+EXCEPTIONS_TO_GENERAL_TESTING = %w[
+  ai_assurance_portfolio_technique
+  business_finance_support_scheme
+  flood_and_coastal_erosion_risk_management_research_report
+  licence_transaction
+  product_safety_alert_report_recall
+  research_for_development_output
+  statutory_instrument
+].freeze
+
+RSpec.feature "Creating a document", type: :feature do
+  shared_context "common setup" do |editor, document_type, document_path, new_document_path|
+    let(:document) { FactoryBot.create(document_type) }
+    let(:content_id) { document["content_id"] }
+    let(:save_button_disable_with_message) { page.find_button("Save as draft")["data-disable-with"] }
+    let(:schema) { document_type.to_s.camelize.constantize.finder_schema }
+
+    before do
+      log_in_as_editor(editor)
+
+      allow(SecureRandom).to receive(:uuid).and_return(content_id)
+      Timecop.freeze(Time.zone.parse(document["public_updated_at"] || "2015-12-03 16:59:13 UTC"))
+
+      stub_any_publishing_api_put_content
+      stub_any_publishing_api_patch_links
+
+      stub_publishing_api_has_content([document], hash_including(document_type: document_type.to_s))
+      stub_publishing_api_has_item(document)
+    end
+
+    scenario "getting to the new document page" do
+      visit document_path
+      click_link "Add another #{schema.document_title}"
+
+      expect(page.status_code).to eq(200)
+      expect(page.current_path).to eq(new_document_path)
+    end
+
+    scenario "with valid data" do
```

Comment:
This reads like a context block (`context "with valid data"`) - I'd then expect an assertion `it` block (or several) to be made further down (e.g. `it "saves the draft"`) to make it clear what we're testing.

Reply:
I'll change the scenario descriptions to read a bit clearer